tests/Integration/Frameworks/Laravel/Version_5_7/tests/Integration/PipeTracingTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Tests\Integration;
+
+use DDTrace\Tests\Integration\Common\SpanAssertion;
+use DDTrace\Tests\Integration\Common\SpanAssertionTrait;
+use DDTrace\Tests\Integration\Common\TracerTestTrait;
+use DDTrace\Tests\Integration\Frameworks\Util\CommonScenariosDataProviderTrait;
+use DDTrace\Tests\Integration\Frameworks\Util\Request\GetSpec;
+use DDTrace\Tests\Integration\Frameworks\Util\Request\RequestSpec;
+use Illuminate\Pipeline\Pipeline;
+use Tests\TestCase;
+
+/**
+ */
+class PipelineTracingTest extends TestCase
```

Comment:
The file name `PipeTracingTest.php` doesn't match the class name. :)

Reply:
Good catch! Fixed it